Skype/SDN/articles/new-features.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: New features in Skype for Business SDN Interface
+title: New features in Skype for Business (Lync Server 2013 | Skype for Business 2015)
  
 ms.assetid: a0ef6ad8-c0a2-425f-b236-dc4907efa22b
 ---
 
 
-# New features in Skype for Business SDN Interface
+# New features in Skype for Business (Lync Server 2013 | Skype for Business 2015)
 
 
   
```

Comment:
Why remove the SDN interface when the information in parentheses fixes the duplicate issue?

Reply:
Adding the info still resulted in an error for duplicate titles...but I will add it back in and include the version number to differentiate the titles. Thank you!